app/models/history.rb

```diff
@@ -28,7 +28,8 @@ class History < ApplicationRecord
     close_contact_edit: 'Close Contact Edit',
     contact_attempt: 'Contact Attempt',
     welcome_message_sent: 'Welcome Message Sent',
-    record_automatically_closed: 'Record Automatically Closed'
+    record_automatically_closed: 'Record Automatically Closed',
+    system_note: 'System Note'
```

Comment:
I think 'Record Edit' is actually the right one for this, since it's a modification of the record. As long as the "who" is the system, that will be clear.

Reply:
Keeping it as System Note makes the DB migration really easy to roll back. Idk if you've looked at the `down` method. Any thoughts? @holmesie

Reply:
how about 'System Edit` ?

Reply:
`System Record Edit` it is!